include/vrv/options.h

```diff
@@ -594,6 +594,7 @@ class Options {
     OptionBool m_mensuralToMeasure;
     OptionDbl m_midiTempoAdjustment;
     OptionDbl m_minLastJustification;
+    OptionDbl m_maxVerticalJustification;
```

Comment:
Please move to the general layout group and rename to `m_justificationMaxVertical` for better grouping (the one above is unfortunately badly placed / named)